apps/cow-fi/theme/ThemeProvider.tsx

```diff
@@ -0,0 +1,16 @@
+import { ThemeProvider as StyledComponentsThemeProvider } from 'styled-components/macro'
+import { baseTheme } from '@cowprotocol/ui'
+import { THEME_MODE } from '@/components/Layout/const'
+import { PropsWithChildren, useMemo } from 'react'
+
+import './types'
+
+function getCowfiTheme() {
+  return baseTheme(THEME_MODE)
+}
+
+export function ThemeProvider(props: PropsWithChildren) {
+  const theme = useMemo(() => getCowfiTheme(), [])
```

Comment:
Don't we need to disable the es-lint rule for this to pass the linter?

Reply:
No, there are no dependencies. `getCowfiTheme` is a pure function